pxr/usdImaging/usdviewq/utils.cpp

```diff
@@ -104,6 +104,15 @@ UsdviewqUtils::PrimInfo::PrimInfo(const UsdPrim &prim, const UsdTimeCode time)
     else
         name = _tokens->root.GetString();
     typeName = prim.GetTypeName().GetString();
+
+    if (prim.HasAuthoredDisplayName())
+    {
+        displayName = prim.GetDisplayName();
+    }
+    else
+    {
+        displayName = "";
+    }
```

Comment:
The HasAuthoredDisplayName check is redundant here; the fallback for GetDisplayName should be the empty string.

Reply:
I ran into an issue early on where it was actually returning `None` when it was not authored.  I can test this again, but there were some tests failing prior to one of the initial commits because of this.

Reply:
Sorry, noticed this was the C++ code, in this case it is redundant.  Will change.

Reply:
Changed to `displayName = prim.GetDisplayName();`